randovania/gui/ui_files/games_tab_samusreturns_widget.ui

```diff
@@ -217,16 +217,24 @@
          </widget>
         </item>
         <item>
-         <widget class="Line" name="line">
-          <property name="orientation">
-           <enum>Qt::Horizontal</enum>
+         <widget class="QTreeWidget" name="hint_locations_tree_widget">
+          <property name="minimumSize">
+           <size>
+            <width>0</width>
+            <height>340</height>
+           </size>
           </property>
+          <column>
+           <property name="text">
+            <string>Location</string>
+           </property>
+          </column>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="hints_label_2">
           <property name="text">
-           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Custom Chozo Seals have also been added to all areas, which provide hints to Metroid DNA. Each seal provides hints randomly between zero and a set maximum. If no hints are provided, a joke hint is provided instead. These custom seals and their maximum amount of hints are as follows:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+           <string>&lt;hr&gt;&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Custom Chozo Seals have also been added to all areas, which provide hints to&lt;span style=&quot; font-weight:600;&quot;&gt; Metroid DNA&lt;/span&gt; is located. Each seal provides hints randomly between zero and a set maximum. If no hints are provided, a joke hint is provided instead. These custom seals and their maximum amount of hints are as follows:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Copy paste error? `which provide hints to Metroid DNA is located` Should be either `where Metroid DNA is located` or just remove the `is located` as it was before?